### PR TITLE
grpc: add exchange field to QueryOrderRequest

### PR DESCRIPTION
**Pull request by narumiruna**

### CODE REVIEW

**Comment by narumiruna** [comment]
@c9s do we need both `id` and `client_order_id` fields in cancel/query order request?

**Comment by narumiruna** [comment]
@c9s do we need both `id` and `client_order_id` fields in cancel/query order request?

**Comment by c9s** [comment]
@narumiruna yes, we need both fields, the exchange implementation will choose order id first (if given), and then the client order id.

File: pkg/pb/bbgo.proto [hunk 1]
```diff
@@ -150,8 +150,9 @@ message CancelOrderResponse {
 }
 
 message QueryOrderRequest {
-    int64 id = 1;
-    int64 client_order_id = 2;
```
Comment (c9s) [inline pkg/pb/bbgo.proto:154]: client order id is usually a string.

File: pkg/pb/bbgo.proto [hunk 2]
```diff
@@ -150,8 +150,9 @@ message CancelOrderResponse {
 }
 
 message QueryOrderRequest {
-    int64 id = 1;
```
Comment (c9s) [inline pkg/pb/bbgo.proto:153]: would prefer using string here, because some exchange use UUID (string).

we can parse the integer from the server side.

Comment (narumiruna) [reply pkg/pb/bbgo.proto:153]: ok

**Review by c9s — APPROVED** [review]

**Pull request merged**